pom.xml

```diff
@@ -129,39 +129,88 @@
+
+              <!-- src/test/bench -->
+
+              <exclude>src/test/bench/query*</exclude>
+
+              <!-- PostgreSQL src/test/regress -->
+              
+              <exclude>src/test/regress/**/*.in</exclude>
+              <exclude>src/test/regress/**/*.mdp</exclude>
+              <exclude>src/test/regress/**/*.xml</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_3chars_line_delimiter.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_big.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_no_line_delimiter.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_out.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_blankfields.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_blankfields_and_nulls.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_correct.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_error_fieldtoolong.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_error_fieldtype.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_error_rejectlimitcrossed.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_error_rejectlimitnotcrossed.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_error_toofewfields.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_error_toomanyfields.tbl</exclude>
+              <exclude>src/test/regress/data/fixedwidth/fixedwidth_small_nulldefined.tbl</exclude>
+              <exclude>src/test/regress/data/nation.tbl</exclude>
+              <exclude>src/test/regress/data/nation_error50.tbl</exclude>
+              <exclude>src/test/regress/data/region.tbl</exclude>
+              <exclude>src/test/regress/expected_statuses</exclude>
+                            
+              <!-- src/test/feature -->
+
+              <exclude>src/test/feature/ExternalSource/data/nation.tbl</exclude>
+              <exclude>src/test/feature/ExternalSource/data/nation_error50.tbl</exclude>
+              <exclude>src/test/feature/ExternalSource/data/region.tbl</exclude>
+              <exclude>src/test/feature/ManagementTool/test_hawq_register_hawq.paq</exclude>
+              <exclude>src/test/feature/ManagementTool/test_hawq_register_hive.paq</exclude>
+              <exclude>src/test/feature/ManagementTool/data/parquet200/dat.paq</exclude>
+              <exclude>src/test/feature/ManagementTool/data/parquet200sum/a.paq</exclude>
+              <exclude>src/test/feature/ManagementTool/data/parquet200sum/b.paq</exclude>
+              <exclude>src/test/feature/ManagementTool/test_hawq_register_data_types.paq</exclude>
```

Comment:
I think we should limit using wildcard for directory, etc, but not for file extensions.